appserver/tests/application/src/test/java/org/glassfish/main/test/app/synchronize/SynchronizeStandaloneInstanceTest.java

```diff
@@ -69,10 +71,8 @@ public void testSynchronization() {
 
     private static File createDeployment() {
         WebArchive webArchive = ShrinkWrap.create(WebArchive.class)
-            .addAsWebInfResource(
-                new File("src/main/resources/org/glassfish/main/test/app/synchronize/sample.json"),
-                "resources/sample.json")
-            .addAsWebResource(new File("src/main/resources/org/glassfish/main/test/app/synchronize/index.html"));
+            .addAsWebInfResource(getTestResourceStatic("sample.json"), "resources/sample.json")
```

Comment:
You don§t need to create a method for that, you can even shorten it to the resource reference. It is also better if you want to filter the file by maven. Perhaps just add json files to resources, like xml files are already there. It is always wrong to touch src/main/resources from the test in runtime (also Eclipse uses different basedir than Maven).

Reply:
I think I got you. Tests should get resources from the classpath instead of from the source directory. I updated the code.